### Coin Flip Expected Value 

The coin is flipped 3 times, we need to get the probabilites:

1. 3 heads

2. 2 heads

3. 1 head

4. no heads 



Set of outcomes:

HHH, HHT, HTT, THH, HTH, HHT, TTT, THT 

$$ p_1 = \frac{1}{8} $$ 

$$ p_2 = \frac{1}{2} $$ 

$$ p_3 = \frac{1}{4} $$ 

$$ p_4 = \frac{1}{8} $$ 

Values are seen in the list below for the outcomes respsectively:



In [12]:
values = [5,-5,-4,4]

In [13]:
probs = [(1/8),(1/2),(1/4),(1/8)]

$$ Expected Value = p1v1 + p2v2 + p3v3 + p4v4 $$ 


In [14]:
E = values[0]*probs[0] + values[1]*probs[1] + values[2]*probs[2] + values[3]*probs[3]
print(E)

-2.375


$$ E \approx -2.375 $$ 

On average, it is expected that you lose 2.375 dollars 